meson.build

```diff
@@ -74,8 +74,9 @@ vkd3d_private_includes  = include_directories(vkd3d_private_includes)
 
 idl_compiler  = find_program('widl')
 idl_generator = generator(idl_compiler,
-  output    : [ '@BASENAME@.h' ],
-  arguments : [ '-h', '-o', '@OUTPUT@', '@INPUT@' ])
+  output       : '@BASENAME@.h',
+  arguments    : [ '-h', '-o', '@OUTPUT@', '-I', '@0@/@1@'.format(meson.current_source_dir(), './subprojects/Vulkan-Headers/include'),
```

Comment:
I assume this is not needed.

Reply:
Yes, once I remove `include` from `.idl`.